reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServerRoutes.java

```diff
@@ -264,18 +264,18 @@ HttpServerRoutes route(Predicate<? super HttpServerRequest> condition,
 			BiFunction<? super HttpServerRequest, ? super HttpServerResponse, ? extends Publisher<Void>> handler);
 
 	/**
-	 * A comparator used to sort HttpRouteHandlers.When call this method,added routes will be sorted use
-	 * this comparator.When add route if already set comparator,routes will be sorted. If you don't want
-	 * to sort routes you can use noCompare() method to recover and keep routes in add order.
+	 * A comparator used to sort routes.When call this method or configure a route,already configured routes will be
+	 * sorted use this comparator.If you don't want to sort routes you can use noComparator() method to recover and
+	 * keep routes in they are configured order.By default there is no sorting and the routes will be interpreted in
+	 * the way they are configured order.
```

Comment:
```suggestion
	 * Use the provided {@link java.util.Comparator} to sort routes, rather than using
	 * declaration order. Routes that were already declared are also impacted by this
	 * change and will be sorted according to the comparator. You can revert to using the
	 * declaration order by calling the {@link #noComparator()} method (which is the default).
```